src/modules/safe-shield/threat-analysis/threat-analysis.constants.ts

```diff
@@ -0,0 +1,65 @@
+import type { CommonStatus } from '@/modules/safe-shield/entities/analysis-result.entity';
+import { COMMON_SEVERITY_MAPPING } from '@/modules/safe-shield/entities/common-status.constants';
+import type { Severity } from '@/modules/safe-shield/entities/severity.entity';
+import type { ThreatStatus } from '@/modules/safe-shield/entities/threat-status.entity';
+
+export const SAFE_VERSION = '1.3.0';
+/**
+ * Severity mapping for threat analysis results.
+ * Maps each contract status to its corresponding severity level.
+ */
+export const SEVERITY_MAPPING: Record<
+  ThreatStatus | CommonStatus,
+  keyof typeof Severity
+> = {
+  ...COMMON_SEVERITY_MAPPING,
+  MALICIOUS: 'CRITICAL',
+  MODERATE: 'WARN',
+  NO_THREAT: 'OK',
+  MASTER_COPY_CHANGE: 'WARN',
```

Comment:
niptick: mastercopy is 1 word (in Safe context)

```suggestion
  MASTERCOPY_CHANGE: 'WARN',
```